### PR TITLE
Adding manager 2

### PR DESCRIPTION
**Pull request by guy9**
Example of running a Manager 2.0 onwards with Docker
@tzach please review

### CODE REVIEW

**Comment by tzach** [comment]
@guy9 look like the PR is missing the docker-compose.yaml, maybe more files

**Comment by guy9** [comment]
I used only a Dockerfile and not docker-compose (similar to the second option in docker hub)

**Comment by tzach** [comment]
So the README is not up to date

**Comment by guy9** [comment]
good catch, I added an extra line in the README file, fixed now

File: manager/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+
+
+### Destroying the Scylla Cluster 
+List the running containers
+```
+docker ps -a
+```
+
```
Comment (tzach) [inline manager/README.md]: You should see the following instances:
```
NAMES
scylla-manager
scylla-manager-db
scylla
minio1
```

**Comment by tzach** [comment]
All seems to work.
Are you planning to add a few commands to demonstrate how to use the manager?

**Comment by guy9** [comment]
I placed a link in the README file to the Docs and to the University website. 
I plan to have more info in the U. lessons with an actual example

**Comment by tzach** [comment]
Thanks
Will be good to add the basic example to the readme as well making it more self contained

**Pull request merged**